### ISSUE TITLE
TypeError: Cannot read property 'asyncComputed' of undefined

### ISSUE THREAD

**Issue by koso00** [alexjoverm/v-runtime-template#14]
i get 

```
TypeError: Cannot read property 'asyncComputed' of undefined
    at Object.vueAsyncComputedInjectedDataFn [as data] (app.js:94221)
    at getKeysFromOptions (app.js:85826)
    at Proxy.render (app.js:85864)
    at VueComponent.Vue._render (app.js:105150)
    at VueComponent.updateComponent (app.js:103394)
    at Watcher.get (app.js:103748)
    at new Watcher (app.js:103737)
    at mountComponent (app.js:103401)
    at VueComponent../node_modules/vue/dist/vue.esm.js.Vue.$mount (app.js:109146)
    at VueComponent../node_modules/vue/dist/vue.esm.js.Vue.$mount (app.js:111545)
```

Every time i render something

**Comment by davek1979** [comment]
Got this problem too, does anyone know a solution?

**Comment by koso00** [comment]
I don't remember well but as far as i remember you must import vue-async-computed from npm just before vue-runtime-template.

**Comment by davek1979** [comment]
Hmm in my case I had to disable vue-async-computed competely to get it to work.

**Comment by H3xx3n** [comment]
Had this issue today. It seems that when the parent component has any asyncComputed and the child component has no data, vue messes up.
Quick fix is to always have data() {return {}} in the child components or set them as functional
![image](https://user-images.githubusercontent.com/16035455/52631201-6edaae80-2ebe-11e9-9a63-20186034efd8.png)

**Comment by iSnow** [comment]
Had this as well, for me this commit in a fork fixed it:

https://github.com/Duncank/v-runtime-template/commit/567dd1a9b3c04f5d39996cfb766c9ee9412a0471

**Comment by alexjoverm** [comment]
Fixed on 1.7.1 6289c88c6812df2037f3e138e35fad75c4fd71cc
